filebeat/crawler/prospector_stdin.go

```diff
@@ -37,7 +37,7 @@ func (p *ProspectorStdin) Run() {
 
 	// Make sure stdin harvester is only started once
 	if !p.started {
-		p.harvester.Start()
+		go p.harvester.Harvest()
```

Comment:
ProspectorStdin is so shallow, we don't even need to run the harvester in a separate go-routine, right?